nomenclature/testing.py

```diff
@@ -27,7 +27,7 @@ def assert_valid_yaml(path: Path):
         )
 
 
-def assert_valid_structure(path: Path):
+def assert_valid_structure(path: Path, mappings="mappings", definitions="definitions"):
```

Comment:
Some type hints would be nice:
```suggestion
def assert_valid_structure(path: Path, mappings: str ="mappings", definitions: str="definitions") -> None:
```